static/app/components/core/layout/container.tsx

```diff
@@ -25,6 +25,12 @@ interface ContainerLayoutProps {
 
   padding?: Responsive<Shorthand<SpacingSize, 4>>;
 
+  /**
+   * Prefer using Flex or Grid gap as opposed to margin.
+   * @deprecated
+   */
+  margin?: Responsive<Shorthand<SpacingSize, 4>>;
```

Comment:
We're seeing `<Flex style={{margin: }}>`. I'd rather support this and mark it as deprecated as opposed to have people use styled.